components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -164,12 +176,13 @@ subroutine crm_physics_register()
    call pbuf_add_field('CRM_T',        'global',dtype_r8,dims_crm_3D,idx)
 
    ! Radiation
-   call pbuf_add_field('CRM_T_RAD',    'physpkg',dtype_r8,dims_crm_rad,idx)
-   call pbuf_add_field('CRM_QV_RAD',   'physpkg',dtype_r8,dims_crm_rad,idx)
-   call pbuf_add_field('CRM_QC_RAD',   'physpkg',dtype_r8,dims_crm_rad,idx)
-   call pbuf_add_field('CRM_QI_RAD',   'physpkg',dtype_r8,dims_crm_rad,idx)
-   call pbuf_add_field('CRM_CLD_RAD',  'physpkg',dtype_r8,dims_crm_rad,idx)
-   call pbuf_add_field('CRM_QRAD',     'global', dtype_r8,dims_crm_rad,idx)
+   call pbuf_add_field('CRM_T_RAD',    'physpkg',dtype_r8,dims_crm_rad,crm_rad_idx_t)
```

Comment:
I think it'd be better to name the indices like the variables they reference, in keeping with the precedent set in other places in the code. I.e., `crm_t_rad_idx` instead of `crm_rad_idx_t` to refer to `CRM_T_RAD`.

Reply:
yea, I figured you would say that. I wish we had used the "crm_rad" prefix to begin with. I'll change it to match.

Reply:
done.